src/ar/MarkerSet.cpp

```diff
@@ -105,185 +105,27 @@ bool MarkerSet::getMarkerByMappedID(int mapped_id, MarkerPattern& out) const {
 namespace Markers {
 
 // Constants - please note that these are only used internally.
-/** The number of ALVAR markers stored in the data array. */
-constexpr size_t ALVAR_COUNT = 18;
-/** The size in pixels of the data region of an ALVAR marker. */
-constexpr size_t ALVAR_BIT_SIZE = 5;
-/** The border size in pixels of one side of an ALVAR marker. */
-constexpr size_t ALVAR_BORDER_SIZE = 2;
-/** The physical size (in mm) of an ALVAR marker. */
-constexpr float ALVAR_PHYS_SIZE = 0.2;
-/** The bit array size for one ALVAR marker, equal to the bit size squared. */
-constexpr size_t BIT_ARR_SIZE = ALVAR_BIT_SIZE * ALVAR_BIT_SIZE;
-
 /** The size in pixels of the data region of an ARUco marker. */
 constexpr size_t ARUCO_BIT_SIZE = 4;
 /** The border size in pixels of one side of an ARUco marker. */
 constexpr size_t ARUCO_BORDER_SIZE = 1;
 /** The physical size (in mm) of an ARUco marker. */
 constexpr float ARUCO_PHYS_SIZE = 0.1; // we don't know this yet, change later
```

Comment:
This should probably be `0.15`, as we now know the size of the markers will be 15cm (20cm, minus 2.5cm of whitespace on both sides)
```suggestion
constexpr float ARUCO_PHYS_SIZE = 0.15; // we don't know this yet, change later
```